lib/katello/tasks/upgrades/4.8/fix_incorrect_providers.rake

```diff
@@ -0,0 +1,32 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '4.8' do
+      desc "Fix custom products incorrectly assigned a Red Hat provider"
+      task :fix_incorrect_providers => ["dynflow:client", "katello:check_ping"] do
```

Comment:
Do I need the `katello:check_ping` here? it doesn't seem to work anyway (complains about foreman-tasks..)

Reply:
I think you need `environment`

https://github.com/Katello/katello/blob/master/lib/katello/tasks/upgrades/4.4/publish_import_cvvs.rake#L5